layers/meta-resin-raspberrypi/recipes-kernel/linux/linux-raspberrypi_4.4.bbappend

```diff
@@ -1,4 +1,4 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi-4.1:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi-4.4:"
```

Comment:
Are there any patches left in this directory? I think not and we can remove it completely.